fastlane/metadata/fr-FR/release_notes.txt

```diff
@@ -0,0 +1,3 @@
+Résolution d’un bogue qui affectait l’apparence de l’extension Partage.
+Résolution de la prise en charge RTL sur les écrans d’authentification.
+Résolution de la prise en charge RTL dans l’éditeur d’étiquettes.
```

Comment:
These are from 4.20.0. I'm guessing they weren't yet localized when I run the script at the time of submitting it?

The same goes for the `id` and `ru` versions.